tools/msstitch/msspercolator_merge.xml

```diff
@@ -0,0 +1,66 @@
+<tool id="msspercolator_merge" name="Merge Percolator" version="1.1">
```

Comment:
tool version is changing every now and then :) Maybe we should macrofy them as well and use 2.5 as the msstich version is doing?